source/reference/mongosync.txt

```diff
@@ -176,6 +176,73 @@ Global Options
 
    Prints ``mongosync`` version information to stdout.
 
+.. _c2c-cli-OIDC:
+
+Using mongosync with OpenID Connect
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+.. include:: /includes/mongosync-and-OIDC.rst
+
+In the connection string, set :urioption:`authMechanism` to
+``MONGODB-OIDC``. Set :urioption:`authMechanismProperties` as needed.
+For example, to connect to a MongoDB cluster running on:
+
+- Microsoft Azure, set ``authMechanismProperties`` to
+  ``ENVIRONMENT=azure``
+- Google Cloud Platform, set ``authMechanismProperties`` to
+  ``ENVIRONMENT=gcp``
+
+For more details about connection strings, see :ref:`mongodb-uri` and
```

Comment:
Shouldn't this link the equivalent of https://deploy-preview-9438--mongodb-internal.netlify.app/reference/connection-string-options/#authentication-options so that they can see more details about it?

Reply:
The latter link is specific to atlas, so we should make that clear. Like this might be refactors to:

For more details about configuring OIDC in connections strings, see https://deploy-preview-9438--mongodb-internal.netlify.app/reference/connection-string-options/#authentication-options
For more details setting up workload identity federation for OIDC in atlas, see </workforce-oidc/#std-label-oidc-authentication-workforce/>`